be/src/connector/hive_connector.cpp

```diff
@@ -519,6 +519,12 @@ Status HiveDataSource::get_next(RuntimeState* state, ChunkPtr* chunk) {
     return Status::OK();
 }
 
+const std::string HiveDataSource::get_custom_coredump_msg() const {
+    const std::string path = !_scan_range.relative_path.empty() ? _scan_range.relative_path : _scan_range.full_path;
+    return strings::Substitute("Hive file path: $0, partition id: $1, length: $2, offset: $3", path,
```

Comment:
file path with Hive

Reply:
path actually contains partition information

Reply:
Only the iceberg table has `_scan_range.full_path`, otherwise only set for `relative_path`.